### DISCUSSION TITLE
Add URQL Nuxt 3 Example to Repo

### DISCUSSION THREAD

**Discussion by iamrobreynolds** [urql-graphql/urql#2045] [Ideas]
I'm getting a sense of frustration from other Vue/Nuxt devs (myself included) with other GraphQL clients and I'm seeing more inquiries and curiosity about alternatives. URQL seems to be the one garnering the most attention. That being said, is there a plan to develop a URQL Nuxt module? If not, I'm casting a vote. If still not, would FormidableLabs be willing to provide docs and/or nuxt-example in URQL repo on how to implement URQL into a Nuxt 3 app?

**Comment by lewebsimple** [comment]
I'm working on a [Nuxt3 module for urql](https://github.com/lewebsimple/nuxt3-graphql) but it's very much a work in progress.
One of the difficulty lies in generalizing all the possible configurations (i.e. graphcache, etc).
Also, initializing the client from the plugin triggers a warning in production concerning the use* outside of setup / lifecycle hooks.

**Comment by innovaweb-dev** [comment]
An example to use Urql with Nuxt3 without additional module.
https://github.com/bicouy0/nuxt3-urql
